app_dart/bin/server.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:math';
 
 import 'package:appengine/appengine.dart';
 import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/request_handlers/get_release_branches.dart';
```

Comment:
This should be exported in lib/cocoon_service.dart instead of directly imported in bin/

Reply:
Very true, moved.